src/Plugin/OgFields/AccessField.php

```diff
@@ -22,61 +19,60 @@ class AccessField extends OgFieldBase implements OgFieldsInterface {
   /**
    * {@inheritdoc}
    */
-  public function fieldDefinition() {
-    return FieldStorageConfig::create(array(
-      'field_name' => OG_DEFAULT_ACCESS_FIELD,
-      'entity_type' => $this->getEntityType(),
-      'type' => 'list_integer',
-      'cardinality' => 1,
-      'settings' => array(
-        'allowed_values' => array(
+  public function getFieldStorageConfigBaseDefinition(array $values = array()) {
+    $values = [
+      'cardinality' => FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED,
+      'settings' => [
+        'allowed_values' => [
           0 => 'Use default roles and permissions',
           1 => 'Override default roles and permissions',
-        ),
+        ],
         'allowed_values_function' => '',
-      ),
-    ));
+      ],
+      'type' => 'list_integer',
+    ];
+
+    return parent::getFieldStorageConfigBaseDefinition($values);
   }
 
   /**
    * {@inheritdoc}
    */
-  public function instanceDefinition() {
-    return FieldConfig::create(array(
-      'label' => $this->t('Group roles and permissions'),
+  public function getFieldConfigBaseDefinition(array $values = array()) {
+    return [
+      'default_value' => [0 => ['value' => 0]],
       'description' => $this->t('Determine if group should use default roles and permissions.'),
-      'default_value' => array(0 => array('value' => 0)),
-      'display_label' => 1,
+      'display_label' => TRUE,
+      'label' => $this->t('Group roles and permissions'),
       'required' => TRUE,
-      'field_name' => OG_DEFAULT_ACCESS_FIELD,
-      'entity_type' => $this->getEntityType(),
-      'bundle' => $this->getBundle(),
-    ));
+    ];
+
+    return parent::getFieldConfigBaseDefinition($values);
```

Comment:
OK, from looking at the same implementation in `AudienceField` it is the intention to let the parent handle it.